dll/win32/msgina/lang/ro-RO.rc

```diff
@@ -164,11 +164,11 @@ BEGIN
     LTEXT "", IDC_REASON_DESCRIPTION, 39, 184, 190, 27
     LTEXT "Co&mentariu:", IDC_STATIC, 39, 214, 80, 8
     EDITTEXT IDC_REASON_COMMENT, 39, 224, 190, 35, ES_MULTILINE | WS_VISIBLE | WS_TABSTOP
-    DEFPUSHBUTTON "Con&firmă", IDOK, 150, 276, 55, 14, WS_GROUP
-    PUSHBUTTON "A&nulează", IDCANCEL, 211, 276, 55, 14
+    DEFPUSHBUTTON "O&K", IDOK, 150, 276, 55, 14, WS_GROUP
```

Comment:
It's very unlikely that Windows sometimes uses the &OK and sometimes the O&K. I bet MS would try to avoid such a thing. I do dare to claim that you did not actually double-check the Romanian Windows here. I do request a screenshot of your Romanian Windows for this one before I will undo my rejection.

Reply:
I fixed the &OK and the O&K issues. Now it's the same everywhere, please recheck. Unfortunately, I cannot send you any screenshot, as I cannot find msgina.dll in my computer.

Reply:
I checked both %systemroot%\system32 and %systemroot%\SysWOW64 and I got nothing related to this file.

Reply:
I do confirm that The collisions are gone now after switching to O&K everywhere. I do still insist on a screenshot. Download a Romanian 2k3 if you don't have it on your current more modern Windows.